packages/eslint-plugin-lit-a11y/lib/rules/scope.js

```diff
@@ -31,12 +33,31 @@ const ScopeRule = {
 
           analyzer.traverse({
             enterElement(element) {
-              if (element.name !== 'th' && elementHasAttribute(element, 'scope')) {
+              if (
+                element.name !== 'th' &&
+                !element.name.includes('-') &&
+                elementHasAttribute(element, 'scope')
+              ) {
                 const loc = analyzer.getLocationForAttribute(element, 'scope');
                 context.report({
                   loc,
                   message: 'The scope attribute may only be used on <th> elements.',
                 });
+              } else if (
+                (element.name === 'th' || element.name.includes('-')) &&
+                elementHasAttribute(element, 'scope') &&
+                !validScopeValues.includes(element.attribs.scope)
```

Comment:
Custom-elements may implement a `scope` attribute with a different API than the native scope attribute, for authentication for example. 

So for custom elements, we should simply allow the scope attribute to be present, and allow it to have any value, since it's not related to the native `scope` attribute that is used on the `th` element 🙂

Reply:
Yep, totally makes sense.
I have change it now to validate the values of scope only if it is a `th`